ios/FluentUI/Label/Label.swift

```diff
@@ -98,9 +97,6 @@ open class Label: UILabel {
     }
 
     private func initialize() {
-        // textColor is assigned in super.init to a default value and so we need to reset our cache afterwards
-        _textColor = nil
-
```

Comment:
Seems like we added this for some reason. Were you able to track down why and make sure we aren't regressing whatever behavior this was trying to fix?